VocaDbModel/Service/AdminService.cs

```diff
@@ -594,6 +594,25 @@ public class AdminService : ServiceBase {
 
 		}
 
+		public void UpdateNormalizedEmailAddresses() {
+
+			PermissionContext.VerifyPermission(PermissionToken.Admin);

```

Comment:
What should I pass for the argument here?

Reply:
This is ok.